packages/react-kit/src/components/Footer/props.ts

```diff
@@ -0,0 +1,24 @@
+export type FooterProps = Readonly<{
+	/**
+	 * A logo component to render in the footer.
+	 * @remarks should be wrapped in a link to the homepage or other relevant page.
+	 * @example
+	 * ```tsx
+	 * export const Logo = ({ exampleLogo }: { exampleLogo: string }) => {
+	 * 	// use the link and image component from your framework if applicable.
+	 * 	return (
+	 * 		<img src={exampleLogo} width="48" height="48" alt="Example Logo" />
+	 * 	);
+	 * };
+	 * ```
+	 *
+	 */
+	Logo:
+		| { src: string; alt: string; href: string; styles?: string }
+		| React.ReactElement
+		| JSX.Element;
+	/**
+	 * Link used for company website.
+	 */
+	Link: string;
```

Comment:
Small thing but this should probs be lowercase. I wonder if we should make this more generic in case someone doesn't want to put a link here, but we can save that for a future enhancement.